README.md

```diff
@@ -23,12 +23,14 @@ Copy plugin files to your plugin's directory. Use the following blueprint anywhe
 Add the following blueprint to wherever you would like the navigation field to appear.
 
 ```yaml
-navigation:
-  label: Navigation
-  type: navigation
-  levels: 5
-  help: Description of menu or where it is used
-  width: 1/2
+fields:
+  fields:
```

Comment:
Duplicated :))

Reply:
That's embarrassing :p sorry for the inconvenience!